specification/resources/resource-manager/Microsoft.Authorization/stable/2019-09-01/policyAssignments.json

```diff
@@ -699,17 +705,30 @@
         },
         "type": {
           "type": "string",
-          "description": "The identity type.",
+          "description": "The identity type. This is the only required field when adding a system assigned identity to a resource.",
           "enum": [
             "SystemAssigned",
             "None"
           ],
           "x-ms-enum": {
             "name": "ResourceIdentityType",
-            "modelAsString": false
+            "modelAsString": false,
+            "values": [
```

Comment:
What is `values` property? I don't see the definition from [swagger definition](https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum)

Reply:
Provides documentation about each value for help docs + generated code comments.  Its used elsewhere in the repo.  See https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum